src/diffusers/pipeline_utils.py

```diff
@@ -112,6 +113,31 @@ def save_pretrained(self, save_directory: Union[str, os.PathLike]):
             save_method = getattr(sub_model, save_method_name)
             save_method(os.path.join(save_directory, pipeline_component_name))
 
+
+    def to(self, torch_device: Optional[Union[str, torch.device]] = None):
+        modules, _ = self.extract_init_dict(dict(self.config))
```

Comment:
Let's maybe call this `module_names` as these are not actual modules.
```suggestion
        module_names, _ = self.extract_init_dict(dict(self.config))
```

Reply:
Good call, I started using the modules and then forgot to rename the var. Thanks!